docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+```jldoctest dataframe
+julia> eltype.(eachcol(german))
+10-element Vector{DataType}:
+ Int64
+ Int64
+ String
+ Int64
+ String
+ String
+ String
+ Int64
+ Int64
+ String
+```
+
+!!! note
+
+  DataFrames.jl allows to use `Symbol`s (like `:id`) and strings (like `"id"`)
+  for all column indexing operations for convenience. However, using `Symbol`s
+  is slightly faster and should generally be preferred, if not generating them
+  via string manipulation.
+
+To remove all rows and columns from a DataFrame you can use `empty` and `empty!` functions to remove all rows from a DataFrame:
+```jldoctest dataframe
+julia> empty(german)
+0×10 DataFrame
+
+julia> german
+1000×10 DataFrame
+  Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  Int64  String  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0     67  male        2  own      NA               little          ⋯
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> empty!(german)
+0×10 DataFrame
+
+julia> german
+0×10 DataFrame
+```
+
+## Getting basic information about a DataFrame
+
+In this section we will learn about how to get basic information on our `German` DataFrame:
+
+The standard `size` function works to get dimensions of the DataFrame,
+```jldoctest dataframe
+julia> german = CSV.read((joinpath(dirname(pathof(DataFrames)),
+                                 "..", "docs", "src", "assets", "german.csv")),
+                       DataFrame) ;
+```
+
+```jldoctest dataframe
+julia> size(german), size(german, 1), size(german, 2)
+((1000, 10), 1000, 10)
+```
+
+To get the number of Rows or Columns in an AbstractDataFrame `german`, you can use `nrow()` and `ncol()`,
+```jldoctest dataframe
+julia> nrow(german), ncol(german)
```

Comment:
again - do not compress

Reply:
I have not understood the meaning of `do not compress` and how can it be avoided?

Reply:
If you write `nrow(german), ncol(german)` a novice user is likely to not understand that you are constructing a tuple (is this something that is immediately obvious for you?).
It is better to write in two lines first show what `nrow(german)` produces and then what `ncol(german)` produces.

Reply:
Ok, thank you.